From 029e58ef085f7172a8107e4b08a7e4937c1e10c7 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:01:03 -0400
Subject: [fs] btrfs: use right extent item position in send when finding extent clones

Message-id: <1407179109-13714-177-git-send-email-sandeen@redhat.com>
Patchwork-id: 87113
O-Subject: [RHEL7.1 PATCH 176/422] Btrfs: use right extent item position in send when finding extent clones
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Filipe David Borba Manana <fdmanana@gmail.com>

commit a0859c0998605d2dc1b021543398cd84a40589db
Author: Filipe David Borba Manana <fdmanana@gmail.com>
Date:   Wed Feb 5 16:48:55 2014 +0000

    Btrfs: use right extent item position in send when finding extent clones

    This was a leftover from the commit:

       74dd17fbe3d65829e75d84f00a9525b2ace93998
       (Btrfs: fix btrfs send for inline items and compression)

    Signed-off-by: Filipe David Borba Manana <fdmanana@gmail.com>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/send.c b/fs/btrfs/send.c
index bef7ba6..89fefbd 100644
--- a/fs/btrfs/send.c
+++ b/fs/btrfs/send.c
@@ -1288,8 +1288,6 @@ static int find_extent_clone(struct send_ctx *sctx,
   extent_item_pos = logical - found_key.objectid;
  else
   extent_item_pos = 0;
-
- extent_item_pos = logical - found_key.objectid;
  ret = iterate_extent_inodes(sctx->send_root->fs_info,
      found_key.objectid, extent_item_pos, 1,
      __iterate_backrefs, backref_ctx);
-- 
1.7.1